framework/platform/bpl/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ elseif (TARGET_PLATFORM STREQUAL "rdkb")
     file(GLOB_RECURSE bpl_platform_sources ${MODULE_PATH}/rdkb/*.c*)
 
     find_package(ubus REQUIRED)
-    list(APPEND BPL_LIBS ubus nl-3 nl-route-3 bridge dl swpal slibc)
+    list(APPEND BPL_LIBS ubus nl-3 nl-route-3 bridge dl slibc uci)
```

Comment:
Why was swpal removed? is it not needed anymore?

Reply:
Apparently not